test/k8sT/Services.go

```diff
@@ -1084,10 +1082,6 @@ var _ = Describe("K8sServicesTest", func() {
 
 				AfterAll(func() {
 					enableBackgroundReport = true
-					kubectl.DeleteCiliumDS()
-					ExpectAllPodsTerminated(kubectl)
-					// Deploy Cilium as the next test expects it to be up and running
-					DeployCiliumAndDNS(kubectl, ciliumFilename)
```

Comment:
The e2e are failing because the `Supports IPv4 fragments` test is missing this `DeployCiliumAndDNS` (contrary to other tests below).

Reply:
@pchaigno I'm not fully understanding this yet because there is a `DeployCiliumAndDNS` in the overall `BeforeAll()` for all services tests

Reply:
The fragmentation test now passed

Reply:
Paul is correct, some of the options from the previous Cilium deployment make the IPv4 fragment test fail (more investigation TBD, could well be just the CT entry not being the same as what the test expects). I'll add a `DeployCiliumAndDNS()` for this test so we can remove it from the `AfterAll`, and investigate further.